src/nav/full-stack-observability.yml

```diff
@@ -2324,21 +2324,21 @@ pages:
                 path: /docs/mobile-monitoring/mobile-monitoring-ui/usage-pages/versions-analysis
               - title: Monthly uniques report
                 path: /docs/mobile-monitoring/mobile-monitoring-ui/usage-pages/monthly-uniques-report
-      - title: New Relic Mobile Android
+      - title: Android agent
```

Comment:
It's weird. This works in my local build, but gets a 404 in the Gatsby preview. I think probably Gatsby hasn't caught up to some of your changes.

Reply:
Hmm, seems to be working for me now in the Gatsby Cloud. Maybe it just caught up? https://docswebsitedevelop-doc6135newrelicmobilecleanup.gtsb.io/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android/